CovertInfiltration/Src/CovertInfiltration/Classes/XComGameState_CovertInfiltrationInfo.uc

```diff
@@ -38,6 +39,57 @@ var bool bCommsJammingTriggered;
 var int NumEnemiesAtMissionStart;
 var protected array<CharacterGroupKillCount> CharacterGroupsKillTracker;
 
+////////////////////////
+/// Strategy helpers ///
+////////////////////////
+
+static function Update ()
```

Comment:
Please no logic in this class, it's there only to hold data. This logic should go inside DLC info (although give it a more specific name, like 'UpdateRemoveActions')